schema/snapcraft.yaml

```diff
@@ -123,6 +123,11 @@ properties:
             uniqueItems: true
             items:
               type: string
+          aliases:
+            type: array
+            items:
+              type: string
+              pattern: "^[a-zA-Z0-9][-_.a-zA-Z0-9]*$"
```

Comment:
@kyrofa that's ok, whatever we do we need to port it back to snapd, I think we want to allow one letter aliases though (not very common but there is stuff like /usr/bin/X that would be a precedent), here in lot of cases we are trying to accommodate a not too crazy set of upstream already chosen command names,  that's why I didn't try to restrict a lot in the first place